examples/packages/graphql-helix/package.json

```
@@ -18,9 +18,16 @@
     "access": "restricted"
   },
   "dependencies": {
+    "@graphql-authz/core": "1.3.1",
+    "@graphql-tools/schema": "10.0.4",
+    "express": "4.19.2",
+    "graphql": "^16.8.1",
```

```suggestion
    "graphql": "16.8.1",
```

please, also check other places